pyDeltaRCM/utils.py

```diff
@@ -0,0 +1,70 @@
+
+import numpy as np
+
+import time
+import numba
+
+# utilities used in various places in the model and docs
```

Comment:
Seems very similar in intent to the [`shared_tools.py`](https://github.com/DeltaRCM/pyDeltaRCM_WMT/blob/develop/pyDeltaRCM/shared_tools.py) file we have. Any reason they can't be combined? (I have no attachment to either name)

Reply:
yeah this would eliminate the shared_tools file. I have this done in another commit, but haven't pushed it. I'll do some of the jitting again, on top of Eric's changes.

Reply:
It turns out that numba doesn't support jitting classes in the way that we would like to use them (for example, they don't import inheritance) and so the way you've done it seems like the simplest way forward for us.